### ISSUE TITLE
Turn Record article on lobbyist bundling into a Wagtail page

### ISSUE THREAD

**Issue by dorothyyeager** [fecgov/fec-cms#3341]
### Summary

**What we are after:**
We have a good Record article on lobbyist bundling. We also need to update the limit every year for that. It makes sense to turn the article into a H4CC page, and follow the approach used for coordinated party expenditure limits. We need to do this in time for the limit update in early February 2020.

### Completion criteria

- [x] Turn https://www.fec.gov/updates/lobbyist-bundling-disclosure-2019/ into a H4CC page.
- [x] New page mentions 2020 limit. ***Note: do not publish until the Federal Register notice is published or when we get the word***
- [x] Update images to be 2020
- [x] Separate images and related text into new reporting example for candidates
- [x] Determine if we need PAC/party version (we don't)
- [x] Second pair of eyes from Info
- [x] Add to filing reports buckets of PAC/Party/Candidate
- [x] Add new page to redirect list

### Future work
- Update redirects currently linking to article
~- Create related reporting examples. (Will need ticket.) We can use existing images until the reporting examples are created.~

**Comment by kathycarothers** [comment]
Sending out for draft for review.

**Comment by kathycarothers** [comment]
Made updates from first review. Jim made some changes to the images too. Have resent out for another review before sending it to Liz and George to review. Waiting for new figures too.

**Comment by kathycarothers** [comment]
Sent to George and Liz for review.

**Comment by kathycarothers** [comment]
Waiting for the new figure.

**Comment by dorothyyeager** [comment]
Moving to 11.3 since this page has only a simple update to make, and it cannot be made until a related Federal Register notice is published. cc @kathycarothers

**Comment by kathycarothers** [comment]
I am just waiting on the new FR notice in order to update the FR notice links on the page.

**Comment by dorothyyeager** [comment]
Moving to 11.4 when we expect the related Federal Register notice to be published.

**Comment by dorothyyeager** [comment]
We've been asked by the Chair's office to publish the limits tomorrow after she signs the FR notice. The page is ready and waiting for that.

**Comment by kathycarothers** [comment]
The page has been published minus the FR notice. Will update the page once the notice is published.
